app/src/main/kotlin/at/bitfire/davdroid/ui/intro/BatteryOptimizationsPage.kt

```diff
@@ -93,38 +65,34 @@ class BatteryOptimizationsPage: IntroPage {
 
     @Composable
     override fun ComposePage() {
-        IntroPage_FromModel()
+        BatteryOptimizationsPageContent()
     }
 
     @Composable
-    private fun IntroPage_FromModel(
+    private fun BatteryOptimizationsPageContent(
```

Comment:
By the guideline: This method should go into the composable file `BatteryOptimizationsPage.kt`